Lanchonete/Lanchonete/ Features/Menu/Data/MenuManager.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  MenuManager.swift
+//  Lanchonete
+//
+//  Created by vera.dias on 2/4/21.
+//
+
+import Foundation
+
+public class MenuManager {
+    func getSandwichTypes () -> [SandwichType] {
+        return SandwichType.allCases
+    }
+    
+    func getSandwichPrice(type: SandwichType) -> Double {
+        return Sandwich(type: type).price()
+    }
+    
+    func getSandwichName(type: SandwichType) -> String {
+        return type.name
```

Comment:
Same here too, sorry. Not sure if this class is necessary. It's not wrong, but it's accessing a data structure that is visible to the consumers (otherwise the consumer wouldn't know the `type`)